src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -33,6 +33,7 @@
     BATCH_RETRY_RUNNING,
     BATCH_RETRY_ERROR,
     ALL_JOBS,
+    saved_jobs,
```

Comment:
It might be an idea to turn `saved_jobs` into `SAVED_JOBS` and initialise it with the other imported variables.